services/QuillLMS/lib/tasks/report_demo.rake

```diff
@@ -8,4 +8,25 @@ namespace :report_demo do
     name = args[:name] ? args[:name].to_s : nil
     Demo::ReportDemoCreator::create_demo("hello+#{name}@quill.org")
   end
+
+  task :generate_new_data, [:email] => :environment do |_, args|
```

Comment:
Would you add a comment explaining the intention here? Should STDOUT be redirect to a specific file? When should we use this rake task?